app/models/ocelot/package.scala

```diff
@@ -31,6 +32,7 @@ package object ocelot {
   def labelReferences(str: String): List[String] = plSingleGroupCaptures(labelRefRegex, str)
   def labelReference(str: String): Option[String] = plSingleGroupCaptures(labelRefRegex, str).headOption
   def isCurrency(str: String): Boolean = inputCurrencyRegex.findFirstIn(str).fold(false)(_ => true)
```

Comment:
No test coverage for the isCurrency method

Reply:
isCurrency no longer used, removed